src/shared/components/lazyMobXTable/LazyMobXTable.spec.tsx

```diff
@@ -398,6 +398,12 @@ describe('LazyMobXTable', () => {
         clock.uninstall();
     });
 
+    describe('downloadData', () => {
```

Comment:
probably you're working on this

Reply:
that's correct, I'm in the process of mocking up a props.columns and props.data to pass into a LazyMobXTable constructor call

Reply:
This looks good, nice job. Given the amount of boilerplate we'll have to add if we do multiple tests, I think this is good enough. Thank you for adding the test.